services/api/src/messaging/players.service.ts

```diff
@@ -157,9 +160,10 @@ export const successfullyJoined = (
 
 export const playerConnected: Handler = (
   ws,
-  { playerName, id, role }: PlayerConnectedMessage,
+  { playerName, id, role, avatar }: PlayerConnectedMessage,
 ): void => {
   const newPlayerId = id ? id : uuidv4();
+  const avatarUrl = avatar ? avatar : undefined;
```

Comment:
Yes, it is, because without this line, the player object get an avatar property with value of null. And I think we don't want that, if the player does not have an avatar, he/she should not contain an avatar property